.github/workflows/docker.yml

```diff
@@ -1,6 +1,9 @@
 name: Docker
 
-on: [push, pull_request]
```

Comment:
this changes the whole flow to "just" do the deploy to staging. 

I would prefer to keep a flow that validates if the docker image is still correct. 

A deploy to staging could be a separate flow then?

Reply:
Ok sorry - I was not 100% sure what this workflow was for. I'll do add a separate workflow.